main_app/views.py

```diff
@@ -136,10 +143,32 @@ class LogoutPageView(LogoutView):
     pass
 
 
-class UserPageView(TemplateView):
+class UserPageView(LoginRequiredMixin, TemplateView):
     template_name = 'user_page.html'
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['age'] = round((datetime.date(datetime.now()) - self.request.user.birthday).days / 365.25)
         return context
+
+
+class MustPageView(LoginRequiredMixin, TemplateView):
```

Comment:
Can we use `ListView` here?
It's is much better for this kind of page
https://docs.djangoproject.com/en/2.2/ref/class-based-views/generic-display/#listview